servlet-security/pom.xml

```diff
@@ -197,7 +190,7 @@
                                 <feature-pack>
```

Comment:
since you going to do changes, please take the opportunity to change the feature pack specification similar to cloud one: 
`<location>org.wildfly:wildfly-datasources-galleon-pack:${version.pack.datasources}</location>`